docs/products/compute/compute-instances/guides/resize/index.md

```diff
@@ -88,18 +88,22 @@ Downgrading to a plan with less resources may be helpful when looking to reduce
 
 ## Switching to a Different Plan Type
 
-There are quite a few types of Compute Instances available to you, each with their own value propositions and use cases. When resizing your instance, you aren't limited to the current plan type. You are able to switch to any plan across any plan type, including:
+There are multiple Compute Instance options available, each with their own benefits and use cases. When resizing your instance, you are also able to switch to a different plan type that may better suit your workload. For help deciding on a plan type, review the [Choosing a Compute Instance Type and Plan](/docs/products/compute/compute-instances/plans/choosing-a-plan/) guide for advice and a comparison of each plan.
+
+To switch your plan type, follow the instructions outlined within [Resizing a Linode](#resizing-a-linode). When choosing the plan, select the tab that corresponds with your desired plan type.
+
+See below for the different Compute Instance plan types available:
 
 - **Dedicated CPU:** Optimized for CPU-intenseive applications. This plan type is equipped with dedicated vCPU cores, suitable for almost any workload that requires consistently high CPU performance. Use cases include production (and high traffic) websites, e-commerce sites, machine learning, data processing, and much more. See [Dedicated CPU Compute Instances](https://www.linode.com/products/dedicated-cpu/).
 
 - **Shared CPU:** Balancing performance with value. This plan type is a solid foundation for many common use cases, including development, low-traffic websites, or any workload that doesn't require consistent 100% CPU usage. See [Shared CPU Compute Instances](https://www.linode.com/products/shared/).
 
+- **Premium CPU:** Guaranteed hardware for CPU-intensive workloads. Built off of our Dedicated CPU offering, this plan comes equipped with the latest AMD EPYC CPUs to make sure your applications are running on the best available hardware with consistent high performance. Use cases include enterprise-grade production applications, video transcoding, and more. See [Premium CPU Compute Instances](https://www.linode.com/products/premium-cpu/).
```

Comment:
~AMD EPYC CPUs~ [AMD EPYC™](https://www.linode.com/amd/) CPUs

It's up to you if you'd like to include the link, but the trademark symbol is needed.